test/SwooleEmitterTest.php

```diff
@@ -72,7 +72,7 @@ public function testMultipleSetCookieHeaders()
             ->withAddedHeader('Set-Cookie', 'bar=baz')
             ->withAddedHeader(
                 'Set-Cookie',
-                'baz=qux; Domain=somecompany.co.uk; Path=/; Expires=Wed, 09 Jun 2021 10:18:14 GMT; Secure; HttpOnly'
+                'baz=qux; Domain=somecompany.co.uk; Path=/; Expires=Wed, 09 Jun 2021 10:18:14 GMT; Secure; HttpOnly; SameSite=strict'
```

Comment:
Do we have tests for `'strict'`, `'Strict'`, `'lax'`, `'Lax'` and `''`? Or not relevant?

Reply:
Good point, we do not have tests for all cases so far. I'll add these.